README.md

```diff
@@ -255,7 +255,7 @@ make cluster-down
 
 # Releasing
 
-1. Checkout a public branch
+1. Checkout a public branch (e.g. `master` or `release-*`)
 2. Call `make prepare-patch|minor|major` and prepare release notes
```

Comment:
I would add after this line another bullet:
Push to remote repository in order to make it public `git push --set-upstream origin <new-branch-name>`

Reply:
`origin` as remote branch is opinionated. For me it is usually the original repo, for others it is their fork. I don't want to tell people how to open PRs, that depends on their git workflow. Anyway, I'll try to make the description more clear.